src/main/haskell/matching-logic/src/Kore/MatchingLogic/ProofSystem/Minimal.hs

```diff
@@ -175,7 +177,7 @@ instance (IsSignature sig, Eq (Sort sig), Eq (Label sig), Eq var) =>
                 phi1 == Just term1a, phi2 == Just term1b -> Right ()
             _ -> Left (Error [] "not proved")
 
-     -- ^ sigma(before ..,\phi1 \/ \phi2,.. after) <->
+     --   sigma(before ..,\phi1 \/ \phi2,.. after) <->
```

Comment:
Since you removed the ^, could you move this before the thing that it is documenting? The same below.